charts/fint-adapter-sikri/templates/fint-adapter-op-item.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.onePasswordVault.create -}}
```

Comment:
There should be one OnePasswordItem pr. secret

Reply:
could you elaborate on that?